### PR TITLE
Fix the dependency version check

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -87,6 +87,7 @@ org.gradle.parallel = true
 buildCacheRetentionDays = 7
 
 kotlin.code.style = official
+kotlin.stdlib.default.dependency=false
```
Comment (mnonnenmacher) [inline gradle.properties:89]: I think this is worth a code comment.

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (sschuberth) [reply gradle.properties:89]: I was hoping "Git blame" would be enough, but I can repeat the info from there in a code comment if you want.

Comment (mnonnenmacher) [reply gradle.properties:89]: Yes, maybe git blame is enough.

**Review by mnonnenmacher — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (sschuberth) [reply gradle.properties:89]: I had actually added a comment but forgot to push. See https://github.com/oss-review-toolkit/ort/pull/4974.

**Pull request merged**